### ISSUE TITLE
Link to IDL pictogram page on Carbon

### ISSUE THREAD

**Issue by assiroud** [carbon-design-system/carbon#4184]
I think it would be helpful if Carbon pictogram page had a link to the one on IDL that explains color usage and shading. 
https://www.ibm.com/design/language/elements/pictograms/design#color

**Comment by laurenmrice** [comment]
Yes I agree, We should reference and have a link to the Design tab on IDL for now until we revise our content. 👍

**Comment by assiroud** [comment]
Hi Lauren  @laurenmrice. Have you had a chance to add the link to the page? Thanks
